tests/lms/views/test_lti_launches.py

```diff
@@ -1,28 +1,43 @@
-from lms.views.lti_launches import lti_launches
+import unittest.mock as mock
 
 import pytest
 
+from lms.views.lti_launches import lti_launches
 from lms.exceptions import MissingLTILaunchParamError
+from tests.lms.conftest import unwrap
+
+
+# The `lti_launches` view function is wrapped in a series of decorators which
+# handle authorization and creating the user/group for the current course.
+#
+# In these tests we only want to test the view function itself, so extract that
+# from the decorated function.
+#
+# nb. This stack of decorators makes it hard to test. It would be much better to
+# use service method calls to handle the work done by decorators.
```

Comment:
Can we remove the last paragraph of this comment? I'm not sure that view body code calling services is actually how we'll refactor this

Reply:
Done